### PR TITLE
Create Developers.md

### PR DESCRIPTION
**Pull request by ns200310**
According to this issue their is no developers.md  so here is the instruction manual
issue-: https://github.com/fossasia/chat.susi.ai/issues/1563

### CODE REVIEW

File: Developers.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<h1>Instructions for users</h1>
+<h2>how to use Susi Ai</h2>
```
Comment (sk9331657) [inline Developers.md]: @ns200310  Capitalise here.

**Comment by AakashMallik** [comment]
@ns200310 Squash your commits and change the commit message to "Fixes <Issue Number>: <Commit message>"

**Comment by AakashMallik** [comment]
@ns200310 I feel there is a lot of scope for professional wirting here. Also, the manual doesn't  feel very descriptive. 
The title of the md says instruction for **Users** but the name of the md is Developer.md. Please look into this.

**Comment by mariobehling** [comment]
Not really needed and too basic.

**Pull request closed without merging**